java/src/main/java/io/cucumber/java/InvalidMethodException.java

```diff
@@ -3,17 +3,24 @@
 import io.cucumber.core.exception.CucumberException;
 
 import java.lang.reflect.Method;
+import java.util.Objects;
 
-final class InvalidMethodException extends CucumberException {
+final class InvalidMethodException
+    extends CucumberException {
```

Comment:
There seems to be no need for a line break here.

Reply:
habbit, do it without thinking, will change back for cucumber stuff